src/validation_workflow/tar.py

```diff
@@ -0,0 +1,27 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from .download_utils import DownloadUtils
+from .validation import Validation
+
+
+class Tar(Validation, DownloadUtils):
+    @classmethod
+    def get_architectures(cls) -> list:
+        return ["x64", "arm64"]
+
+    @classmethod
+    def download_urls(cls, projects: list, version: str, platform: str) -> None:
+        for project in projects:
+            for package_type in cls.get_architectures():
+                url = f"https://artifacts.opensearch.org/releases/bundle/{project}/{version}/{project}-{version}-{platform}-{package_type}.tar.gz"
+                if Tar.is_url_valid(url):
+                    print(f" Valid URL - {url}")
```

Comment:
Make the return type as boolean. True if successful, else false along with logging. We want the workflow to fail if the validation fails instead of throwing false positive and just logging.